lib/enigma.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './encryptor'
+require_relative './decryptor'
+
+class Enigma
+
+  def encrypt(message, key ='', date='')
```

Comment:
the `encrypt` method can be called with a `key` and a `date` passed in but these two arguments are optional.